src/Argu/project.json

```diff
@@ -0,0 +1,64 @@
+{
+    "version": "3.0.0-beta02",
+    "buildOptions": {
+        "debugType": "portable",
+        "compilerName": "fsc",
+        "define": [
+            "CORE_CLR"
+        ],
+        "compile": {
+            "includeFiles": [
+                "AssemblyInfo.fs",
+                "Types.fs",
+                "Utils.fs",
+                "ConfigReaders.fs",
+                "UnionArgInfo.fs",
+                "PreCompute.fs",
+                "UnParsers.fs",
+                "ParseResult.fs",
+                "Parsers/Common.fs",
+                "Parsers/Cli.fs",
+                "Parsers/KeyValue.fs",
+                "ArgumentParser.fs"
+            ]
+        }
+    },
+    "tools": {
+        "dotnet-compile-fsc": {
+        "version": "1.0.0-preview2-*",
+        "imports": "dnxcore50"
+        }
+    },
+    "frameworks": {
+        "netstandard1.6": {
+            "buildOptions": {
+                "define": [
+                "NETSTANDARD1_5",
+                "NETSTANDARD1_6"
```

Comment:
Yeah maybe being explicit is a good idea anyway (as we saw in the 1.5 -> 1.6 move)...?